src/gui/plugins/plotting/README.md

```diff
@@ -0,0 +1,27 @@
+# Ignition Plotting
```

Comment:
These instructions don't apply now that the plugin is in this repository, right? How about updating the info and moving it to a [tutorial](https://github.com/ignitionrobotics/ign-gazebo/tree/master/tutorials)?